php/src/Webhook.php

```diff
@@ -16,6 +16,12 @@ public function __construct($secret)
         $this->secret = base64_decode($secret);
     }
 
+    public static function fromRaw($secret) {
+        $obj = new self(); 
+        $obj->secret = $secret;
+        return $obj;
+    }
```

Comment:
@tasn does this look right? When I googled PHP's `base64_decode` function, it looks like it returns a string anyways, not `[]byte`. So in PHP at least, we only handle strings, is this correct?

Reply:
I think that's correct, yeah.